datanode/gateway/graphql/schema.graphql

```diff
@@ -4069,6 +4069,8 @@ enum ProposalRejectionReason {
   PROPOSAL_ERROR_INVALID_SLA_PARAMS
   "Perpetual market proposal contained invalid product definition"
   PROPOSAL_ERROR_INVALID_PERPETUAL_PRODUCT
+  "Spot market proposal contains too many position decimal places"
+  PROPOSAL_ERROR_INVALID_POSITION_DECIMAL_PLACES
```

Comment:
Should the error name specify spot?

Reply:
whatever you think is best, at the moment it can only be returned from a spot market